mc-sema/peToCFG/peToCFG.cpp

```diff
@@ -618,6 +632,20 @@ NativeModulePtr readProtoBuf(std::string fName, const llvm::Target *T) {
       externData.push_back(deserializeExtData(ed));
     }
 
+    for(int i = 0; i < serializedMod.offset_tables_size(); i++) {
+        const ::OffsetTable &ot = serializedMod.offset_tables(i);
+
+        std::vector< std::pair<VA,VA> > v;
+
+        for( int j = 0; j < ot.table_offsets_size(); j++) {
+            v.push_back( std::pair<VA,VA> ( ot.table_offsets(j), ot.destinations(j) ) );
```

Comment:
The latter -- I wanted to preserve the original multiple entries in case it was somehow needed later.